src/Components/voting.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import { addVotes } from "../utils/api";
+
+export default function Voting({ votes, review_id }) {
+  const [votesChange, setvotesChange] = useState(0);
+  const [err, setErr] = useState(null);
+
+  function upVote() {
+    setvotesChange((currVotesChange) => currVotesChange + 1);
+    addVotes(review_id, 1).catch((err) => {
+      setvotesChange((currVotesChange) => currVotesChange - 1);
+      setErr("Something went wrong, please try again");
+    });
```

Comment:
This is fantastic! You're optimistically rendering the vote change, and accounting for if it goes wrong by catching the error and reverting the votes!